### PR TITLE
Add Debug button to #lang htdp/asl.

### LINKED ISSUE #177: No debug toolbar button for `#lang htdp/asl`
The menu-based language has one, so `#lang htdp/asl` should have one too.

### PR DESCRIPTION
**Pull request by mikesperber**
Fixes #177.

### CODE REVIEW

**Comment by mikesperber** [comment]
@rfindler This avoids the "two syntax-check buttons" problem.

**Comment by mikesperber** [comment]
This also makes https://github.com/racket/drracket/pull/592 obsolete.

File: htdp-lib/htdp/bsl/reader.rkt [hunk 1]
```diff
@@ -26,15 +26,18 @@
 (define ((make-info options) key default use-default)
   (case key
     [(drscheme:toolbar-buttons)
+     ;; If you want to add more buttons, don't do it this way:
```
Comment (rfindler) [inline htdp-lib/htdp/bsl/reader.rkt:29]: Maybe we should write something like:

```
;; if you want to add more buttons, do it this way only 
;; if they aren't already built and can just be enabled.
```

?

Comment (jbclements) [reply htdp-lib/htdp/bsl/reader.rkt:29]: I had trouble parsing this comment; I think you mean 
"aren't already (built and can just be enabled)", but I initially read it as "(aren't already built) and (can just be enabled)"... which doesn't really make sense, but maybe something like "aren't already built and enable-able" would be less misunderstandable. Oog.

Comment (rfindler) [reply htdp-lib/htdp/bsl/reader.rkt:29]: How's this:

```
;; if you want to add more buttons, the strategy used here might not
;; be the right one to use. If you can simply enable existing buttons,
;; do that instead.
```

Comment (mikesperber) [reply htdp-lib/htdp/bsl/reader.rkt]: @rfindler Done.

**Pull request merged**